contributing/topics/guide-breaking-changes.md

```diff
@@ -391,3 +391,11 @@ Our tests were failing because the Azure API was returning this value as true wh
 ```
 
 There are many ways to accidentally add a breaking change when looking at properties with a Default or lack thereof so extra work needs to be done to confirm what Terraform and the Azure API are returning before deciding how best to incorporate the Default tag.
+
+## Major Release Feature Flag Removal
```

Comment:
Can you add this as a linked section at the top of the guide?

Also the name of this section feels a bit out of place for a guide on breaking changes.. the only suggestion I have is `Post Breaking Change Clean Up` but I'm not sure if that is really better or not.